### PR TITLE
Cdk mirror transformer

### PR DESCRIPTION
**Pull request by arajguha**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: versionedRouter.js [hunk 1]
```diff
@@ -94,14 +103,47 @@ async function handleDest(ctx, version, destination) {
...
         parsedEvent = processDynamicConfig(parsedEvent);
-        let respEvents = await destHandler.process(parsedEvent);
-
+        let respEvents;
+        if (isCdkDestination(destination)) {
+          const cdkResponse = await Executor.execute(
+            event,
+            ConfigFactory.getConfig(destination)
+          );
+          respEvents = await destHandler.process(parsedEvent);
```
Comment (sanpj2292) [inline versionedRouter.js:119]: You can put this below Line 113 & delete the else block

Comment (arajguha) [reply versionedRouter.js]: done

**Comment by arajguha** [comment]
pushed a local image build to `rudderlabs/develop-rudder-transformer`  for quick testing.

**Comment by sanpj2292** [comment]
It is a stale old PR for mirror transformer

**Comment by sanpj2292** [comment]
Closing this as stale PR

**Pull request closed without merging**